rmw_iceoryx_cpp/src/internal/iceoryx_name_conversion.cpp

```diff
@@ -99,6 +99,13 @@ get_name_n_type_from_iceoryx_service_description(
   if (service.find("/msg/") != std::string::npos) {
     // ROS 2.0 Naming
     return std::make_tuple(instance, service);
+  } else if (service.find("Introspection") != std::string::npos) {
+    // iceoryx built-in topic handling
+    std::string delimiter_msg = "_iceoryx/";
+
+    return std::make_tuple(
+      "/" + delimiter_msg + instance + "/" + service + "/" + event,
+      "iceoryx_introspection_msgs/msg/" + event);
```

Comment:
Do we actually have these messages?

Reply:
we don't. Hence the decision to make the topic hidden (leading '_'). 
With https://github.com/ros2/rosbag2/pull/332, rosbag2 won't pick up the hidden topics then by default.

In the future we could re-iterate over this and introduce the actual `introspection_msgs` to parse iceoryx's meta information with all common rostools.

Reply:
Internally we are using the iceoryx introspection to feed the different rostools. I just was thinking it would be nice to be able to `ros2 topic echo` these topics.

Reply:
I think these have to be (de-)serialized specifically as such then, don't they? I don't think it's a real trivial change and I would like to keep this outside of the scope of this PR.